kraken/node_actions/common_node_functions.py

```diff
@@ -39,3 +40,28 @@ def wait_for_unknown_status(node, timeout):
 # Get the ip of the cluster node
 def get_node_ip(node):
     return runcommand.invoke("kubectl get node %s -o jsonpath='{.status.addresses[?(@.type==\"InternalIP\")].address}'" % (node))
+
+def check_service_status(node, service, timeout):
+    ssh = paramiko.SSHClient()
+    ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+    i = 0
+    sleeper = 1
+    while i <= timeout:
+        try:
+            time.sleep(sleeper)
+            i += sleeper            
+            logging.info("Trying to ssh to instance: %s" % (node))
+            connection = ssh.connect(node, username='root', key_filename='/root/.ssh/id_rsa', timeout=800, banner_timeout=400)
```

Comment:
would we be able to add this ssh key parameter as a parameter in the node scenario yaml file